hack/bazel-push-images.sh

```diff
@@ -22,12 +22,14 @@ set -e
 source hack/common.sh
 source hack/config.sh
 
+BAZEL=${BAZEL:-bazel}
```

Comment:
Is this to allow the usage of `bazelisk`? If so, I can recommend symlinking bazelisk with bazel ( had issues with a bash alias due to the wrapped calling of each other, therefore a symlink). That is what I have and it works pretty nice outside of the build container:

```
$ ls -lh /home/rmohr/bin/bazel
lrwxrwxrwx. 1 rmohr rmohr 8 Jan 12 12:00 /home/rmohr/bin/bazel -> bazelisk
```

From the development cycle perspective, all important target except `hack/generate.sh` (quite some code generators left which you would have to pre-install) should work this way.

Reply:
Done